tests/models/paligemma/test_processing_paligemma.py

```diff
@@ -0,0 +1,209 @@
+# Copyright 2023 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import shutil
+import tempfile
+import unittest
+
+import numpy as np
+
+from transformers.testing_utils import require_torch, require_vision
+from transformers.utils import is_vision_available
+
+from ...test_processing_common import ProcessorTesterMixin
+
+
+if is_vision_available():
+    from PIL import Image
+
+    from transformers import (
+        AutoProcessor,
+        PaliGemmaProcessor,
+        is_vision_available,
+    )
+
+
+@require_vision
+class PaliGemmaProcessorTest(ProcessorTesterMixin, unittest.TestCase):
+    processor_class = PaliGemmaProcessor
+
+    def setUp(self):
+        self.tmpdirname = tempfile.mkdtemp()
+        processor = PaliGemmaProcessor.from_pretrained("google/paligemma-3b-pt-224")
```

Comment:
This will indeed cause a gated repo issue. you could rebuild a processor without using this repo, something like 
```suggestion
        image_processor = SiglipImageProcessor.from_pretrained("google/siglip-so400m-patch14-384")
        tokenizer = GemmaTokenizer(SAMPLE_VOCAB, keep_accents=True)
        processor = PaliGemmaProcessor(image_processor=image_processor, tokenizer=tokenizer)
```
Where
`SAMPLE_VOCAB = get_tests_dir("fixtures/test_sentencepiece.model")`
as it is done for `test_tokenization_gemma.py`.

Not sure if that's the nicest way to fix this though, any idea @zucchini-nlp @molbap ?

Reply:
Sorry, what is the issue here? This repo seem to be public no?

Reply:
![image](https://github.com/user-attachments/assets/070a9003-37f6-4a76-974c-c5a8b822ff30)
I suppose it is, with a license to accept?

Reply:
wait, are we talking about 

google/paligemma-3b-pt-224

or 

google/siglip-so400m-patch14-384

but both are accessible even if I am using a firefox private window